src/components/Product/Filters.js

```diff
@@ -0,0 +1,44 @@
+const getSortedData = (state, data) => {
+  switch (state.sortBy) {
+    case "LOW_TO_HIGH":
+      return [...data].sort((a, b) => {
+        return a.price - b.price;
+      });
+
+    case "HIGH_TO_LOW":
+      return [...data].sort((a, b) => {
+        return b.price - a.price;
+      });
+
+    default:
+      return data;
+  }
+};
+
+const getFilteredData = (data, includeOutOfStock, fastDelivery) => {
+  var filteredData = [...data];
```

Comment:
Is this `var` intentional? If not then you could change it to `let` or `const` depending on use-case

Reply:
okay i will replace it, thanks for pointing out